io.github.mmarco94.tambourine.yml

```diff
@@ -0,0 +1,60 @@
+app-id: io.github.mmarco94.tambourine
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '22.08'
+    add-ld-path: .
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: /app/bin/run.sh
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+  - name: tambourine
+    buildsystem: simple
+    build-commands:
+      # ffsampledsp requires libbz2.so.1.0, however Flatpak only provides libbz2.so.1.0.8
+      - ln -s /usr/lib/x86_64-linux-gnu/libbz2.so.1.0.8 /app/jre/lib/libbz2.so.1.0
```

Comment:
how is this gonna work on `aarch64`?

Reply:
also this is bound to fail, there is a reason with soname have versions.

Reply:
> how is this gonna work on aarch64?

It isn't. Unfortunately one of my dependency is only compiled for x86. Is there a way to restrict the manifest to be x86 only?

Reply:
> also this is bound to fail, there is a reason with soname have versions.

This is adding a symlink from a more to a less specific `.so` (`1.0.8` -> `1.0`), so it should work

Reply:
why not building from source instead?

Reply:
It's way out of scope for my project. 
I've opened https://github.com/hendriks73/ffsampledsp/issues/17 so they can fix it upstream